### DISCUSSION TITLE
Big Family version: Multiple Meal Plans sharing recipes

### DISCUSSION THREAD

**Discussion by rickitaly** [mealie-recipes/mealie#2093] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
In a big family, i would like to have the possibility to give the app for usage also to my relatives, and we would like all to share the same recipes database, it could be our mom or grandma also to share their "heritage" to the whole family 

**(Optional): Suggest A Solution**
i would just add multiple meal plans in first release, no problem with user restrictions over visibility or edit. Then of course in next steps we could think about linking groups to visibility of different shopping lists or meal plans.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by hay-kot** [comment]
I generally like this idea and think we could add it. 

The way I see it is that we should add a Household Subgroup to Groups and then associate Shopping Lists and MealPlans with Households. 

I'm not sure how much separation is necessary there to make it useful, but I think we ought to be able to tack this on. 

---

I or someone else would need to do a more detailed write up of what's required on a code/database level and then we can get it added as a task in the issue tracker.

**Comment by PartemImperium** [comment]
Has there been any progress on this? Like most I set mealie up and it works great for just me. However I would love to share it with friends and family and have shared recipes and this seems like the best solution I see (both thinking about it myself and looking over other feature requests).

**Comment by TheDireOne** [comment]
I too very much want this feature!
